rock_paper_scissors/rps.py

```diff
@@ -3,7 +3,20 @@
 import sys
 
 def rock_paper_scissors(n):
-  pass 
+  def recursiveAdd(recursion,handslist=[[]]):
+    if recursion == n:
+      return handslist
+    results = []    
+    for singleResult in handslist.copy():
+      rockList = singleResult.copy()+ ["rock"]
+      paperList = singleResult.copy()+ ["paper"]
+      sissorsList = singleResult.copy()+["scissors"]
+      results.append(rockList)
+      results.append(paperList)
+      results.append(sissorsList)
+    return recursiveAdd(recursion+1,results)
+
+  return  recursiveAdd(0)
```

Comment:
It took me a few minutes to figure out how you did this but it looks like you're starting from 0 and incrementing until you get to n, and you're using copy to avoid mutating the original list each time this is run. Nice work!